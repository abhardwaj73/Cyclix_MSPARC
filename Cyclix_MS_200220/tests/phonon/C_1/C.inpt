CELL: 20.000000000000000 0.392699081698724 8.001
FD_GRID: 100 24 42
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
BC: D C P
FD_ORDER: 12
EXCHANGE_CORRELATION: LDA_PZ
CHEB_DEGREE: 60
TOL_SCF: 1e0
SMEARING: 0.001
ELEC_TEMP_TYPE: fd
QPOINT_GRID: 0 0 0
