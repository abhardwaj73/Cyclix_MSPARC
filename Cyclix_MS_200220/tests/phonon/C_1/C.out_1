***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 22-Jan-2022  Start time: 16:09:54                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 8.001000 
FD_GRID: 50 12 21
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 2 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 60
NSTATES: 7
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/phonon/C_1/C.out_1
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.400000 (Bohr)
Mesh spacing in y-direction        :  0.384948 (Bohr)
Mesh spacing in z direction        :  0.381000 (Bohr)
Number of symmetry adapted k-points:  2
Output printed to                  :  ../tests/phonon/C_1/C.out_1
Total number of atom types         :  1
Total number of atoms              :  1
Total number of electrons          :  4
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.32 0.63 7.24
Number of atoms of type 1          :  1
Estimated total memory usage       :  46.87 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.4421924527E+00        1.223E-01        5.605
2            -5.4710301027E+00        6.977E-02        1.853
3            -5.4732670299E+00        3.984E-02        1.756
4            -5.4740554809E+00        1.735E-02        1.854
5            -5.4741117323E+00        1.103E-02        1.729
6            -5.4741071534E+00        3.649E-03        1.773
7            -5.4741226669E+00        2.160E-03        1.782
8            -5.4741235040E+00        8.400E-04        1.747
9            -5.4741245525E+00        4.858E-04        1.797
10           -5.4741251700E+00        4.192E-04        1.662
11           -5.4741257378E+00        4.115E-05        1.693
12           -5.4741256468E+00        1.792E-05        1.664
13           -5.4741255918E+00        1.020E-05        1.640
14           -5.4741255950E+00        3.235E-06        1.672
15           -5.4741256061E+00        6.452E-07        1.566
Total number of SCF: 15    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.4741256061E+00 (Ha/atom)
Total free energy                  : -5.4741256061E+00 (Ha)
Band structure energy              : -1.6144739948E+00 (Ha)
Exchange correlation energy        : -1.3894384898E+00 (Ha)
Self and correction energy         : -1.0958669942E+01 (Ha)
Entropy*kb*T                       : -7.6301158474E-04 (Ha)
Fermi level                        : -2.5545272585E-01 (Ha)
Average force                      :  2.4299875675E-02 (Ha/Bohr)
Maximum force                      :  2.4299875675E-02 (Ha/Bohr)
Time for force calculation         :  0.306 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  31.644 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
