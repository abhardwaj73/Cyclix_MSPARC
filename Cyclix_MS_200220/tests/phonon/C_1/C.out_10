***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 23-Jan-2022  Start time: 13:29:38                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 8.001000 
FD_GRID: 50 12 21
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 60
NSTATES: 15
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/phonon/C_1/C.out_10
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.400000 (Bohr)
Mesh spacing in y-direction        :  0.384948 (Bohr)
Mesh spacing in z direction        :  0.381000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/phonon/C_1/C.out_10
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  16
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C_local.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.32 0.63 7.24
Number of atoms of type 1          :  4
Estimated total memory usage       :  41.77 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -6.6302986213E+00        2.802E-01        3.474
2            -6.6624958596E+00        8.362E-02        1.704
3            -6.6705220559E+00        3.554E-02        2.440
4            -6.6727873846E+00        1.982E-02        1.876
5            -6.6732515165E+00        1.344E-02        1.849
6            -6.6734013327E+00        2.127E-03        1.798
7            -6.6734165239E+00        5.374E-04        1.878
8            -6.6734194246E+00        3.069E-04        1.847
9            -6.6734197345E+00        2.052E-04        1.831
10           -6.6734199859E+00        6.419E-05        1.843
11           -6.6734199248E+00        3.363E-05        1.850
12           -6.6734199425E+00        6.998E-06        1.791
13           -6.6734199392E+00        5.792E-06        1.752
14           -6.6734199547E+00        1.944E-06        1.758
15           -6.6734199491E+00        8.222E-07        1.694
Total number of SCF: 15    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -6.6734199491E+00 (Ha/atom)
Total free energy                  : -2.6693679797E+01 (Ha)
Band structure energy              : -1.2119877120E+01 (Ha)
Exchange correlation energy        : -7.2148395045E+00 (Ha)
Self and correction energy         : -4.3810632043E+01 (Ha)
Entropy*kb*T                       : -1.6050052955E-08 (Ha)
Fermi level                        : -1.3359499307E-01 (Ha)
Average force                      :  3.1576182409E-01 (Ha/Bohr)
Maximum force                      :  3.2297281000E-01 (Ha/Bohr)
Time for force calculation         :  0.849 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  488.269 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
