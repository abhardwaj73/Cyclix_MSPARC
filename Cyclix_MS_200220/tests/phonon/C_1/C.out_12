***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 26-Jan-2022  Start time: 10:39:58                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 8.001000 
FD_GRID: 50 12 21
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 60
NSTATES: 15
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E+00
TOL_POISSON: 1.00E-02
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.40
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/phonon/C_1/C.out_12
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.400000 (Bohr)
Mesh spacing in y-direction        :  0.384948 (Bohr)
Mesh spacing in z direction        :  0.381000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/phonon/C_1/C.out_12
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  16
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.32 0.63 7.24
Number of atoms of type 1          :  4
Estimated total memory usage       :  41.77 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.7498241176E+00        1.123E+00        3.540
2            -5.6291727193E+00        4.788E-01        1.338
3            -5.6450291430E+00        2.919E-01        1.351
Total number of SCF: 3     
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.6450291430E+00 (Ha/atom)
Total free energy                  : -2.2580116572E+01 (Ha)
Band structure energy              : -1.4106919551E+01 (Ha)
Exchange correlation energy        : -7.1848239034E+00 (Ha)
Self and correction energy         : -4.3810617659E+01 (Ha)
Entropy*kb*T                       : -3.4192356314E-10 (Ha)
Fermi level                        : -6.0738761246E-01 (Ha)
Average force                      :  3.0013718242E-01 (Ha/Bohr)
Maximum force                      :  3.1267862865E-01 (Ha/Bohr)
Time for force calculation         :  0.863 (sec)
