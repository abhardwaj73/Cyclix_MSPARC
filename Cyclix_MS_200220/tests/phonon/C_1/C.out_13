***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 26-Jan-2022  Start time: 10:40:48                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 8.001000 
FD_GRID: 50 12 21
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 60
NSTATES: 10
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E+00
TOL_POISSON: 1.00E-02
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.40
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/phonon/C_1/C.out_13
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.400000 (Bohr)
Mesh spacing in y-direction        :  0.384948 (Bohr)
Mesh spacing in z direction        :  0.381000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/phonon/C_1/C.out_13
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.32 0.63 7.24
Number of atoms of type 1          :  2
Estimated total memory usage       :  39.81 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.6263887634E+00        1.159E-01        2.542
2            -5.6466395578E+00        5.975E-02        1.129
3            -5.6307722760E+00        3.138E-02        1.117
Total number of SCF: 3     
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.6307722760E+00 (Ha/atom)
Total free energy                  : -1.1261544552E+01 (Ha)
Band structure energy              : -3.0490561677E+00 (Ha)
Exchange correlation energy        : -2.7791023339E+00 (Ha)
Self and correction energy         : -2.1905132853E+01 (Ha)
Entropy*kb*T                       : -8.2239165661E-11 (Ha)
Fermi level                        : -2.1441269323E-01 (Ha)
Average force                      :  1.2064155848E-01 (Ha/Bohr)
Maximum force                      :  1.3559800082E-01 (Ha/Bohr)
Time for force calculation         :  0.373 (sec)
