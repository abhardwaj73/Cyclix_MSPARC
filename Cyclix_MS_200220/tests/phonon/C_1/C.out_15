***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 26-Jan-2022  Start time: 16:17:26                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 8.001000 
FD_GRID: 100 24 42
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 60
NSTATES: 15
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E+00
TOL_POISSON: 1.00E-02
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-02
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/phonon/C_1/C.out_15
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.200000 (Bohr)
Mesh spacing in y-direction        :  0.192474 (Bohr)
Mesh spacing in z direction        :  0.190500 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/phonon/C_1/C.out_15
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  16
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  1.59 0.13 1.52
Number of atoms of type 1          :  4
Estimated total memory usage       :  330.89 MB
