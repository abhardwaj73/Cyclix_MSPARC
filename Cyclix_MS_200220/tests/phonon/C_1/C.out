***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 22-Jan-2022  Start time: 16:08:10                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 8.001000 
FD_GRID: 50 12 21
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 16 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 60
NSTATES: 15
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/phonon/C_1/C.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.400000 (Bohr)
Mesh spacing in y-direction        :  0.384948 (Bohr)
Mesh spacing in z direction        :  0.381000 (Bohr)
Number of symmetry adapted k-points:  9
Output printed to                  :  ../tests/phonon/C_1/C.out
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  16
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.32 0.63 7.24
Number of atoms of type 1          :  4
Estimated total memory usage       :  141.78 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.6924391939E+00        2.056E-01        15.424
2            -5.7072073585E+00        1.117E-01        3.843
